### ISSUE TITLE
False positive for Lint/UselessAssignment

### ISSUE THREAD

**Issue by elia** [Shopify/erb-lint#126]
I'm getting 

```
Lint/UselessAssignment: Useless assignment to variable - `rows`.
In file: app/views/spree/admin/vanity_urls/_form_row.html.erb:18

Lint/UselessAssignment: Useless assignment to variable - `autogrow`.
In file: app/views/spree/admin/vanity_urls/_form_row.html.erb:19
```

for this code (extracted from a larger file):

```erb
<td>
  <%= f.label :message %>
  <br />
  <% rows = record.message.to_s.count("\n") + 1 %>
  <% autogrow = 'this.setAttribute("rows", this.value.match(/$|\n/g).length)' %>
  <%= f.text_area :message, disabled: record.persisted?, rows: rows, onkeyup: autogrow %>
</td>
```

**Comment by EiNSTeiN-** [comment]
Hi @elia, this is normal. Each ruby statement (between erb tags `<% ... %>`) is parsed and analyzed independently of each other. Condiser disabling this cop in your erb-lint configuration.

**Comment by elia** [comment]
gotcha, thanks, maybe it's worth adding a mention of that to the readme, would you mind if I send a PR?

**Comment by EiNSTeiN-** [comment]
That would be appreciated
